ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_filter.proto

```diff
@@ -56,6 +56,16 @@ message InclusiveFilters {
   // the available packages at the time of the query.
   // Optional
   repeated InterfaceFilter interface_filters = 2;
+
+  // A collection of templates for which the payload will be included in the
+  // ``create_arguments`` of a matching ``CreatedEvent``.
+  // SHOULD NOT contain duplicates.
+  // In contrast to the ``template_ids`` field, one can specify whether the blob should be provided
+  // alongside the create arguments.
+  // All ``template_id`` inside need to be valid: corresponding template should be defined in one of
+  // the available packages at the time of the query.
+  // Optional
+  repeated TemplateFilter template_flters = 3;
```

Comment:
```suggestion
  repeated TemplateFilter template_filters = 3;
```

Reply:
I had that corrected in a subsequent commit, but couldn't merge due to borked repo